jupyterlite_sphinx/jupyterlite_sphinx.js

```diff
@@ -180,29 +213,51 @@ window.loadTryExamplesConfig = async (configFilePath) => {
       tryExamplesGlobalMinHeight = parseInt(data.global_min_height);
     }
 
-    // Disable interactive examples if file matches one of the ignore patterns
-    // by hiding try_examples_buttons.
-    Patterns = data.ignore_patterns;
+    // Selectively enable interactive examples if file matches one of the ignore patterns
+    // by un-hiding try_examples_buttons with a smooth transition
+    const Patterns = data.ignore_patterns || [];
+    let shouldShowButtons = true;
+
     for (let pattern of Patterns) {
       let regex = new RegExp(pattern);
       if (regex.test(currentPageUrl)) {
-        var buttons = document.getElementsByClassName("try_examples_button");
-        for (var i = 0; i < buttons.length; i++) {
-          buttons[i].classList.add("hidden");
-        }
+        shouldShowButtons = false;
         break;
       }
     }
+
+    if (shouldShowButtons) {
+      const buttons = document.getElementsByClassName(
+        "try_examples_button hidden",
+      );
+      requestAnimationFrame(() => {
+        for (let button of buttons) {
+          button.style.opacity = "0";
+          button.classList.remove("hidden");
+          button.classList.add("fade-in");
+          requestAnimationFrame(() => {
+            button.style.opacity = "1";
+          });
+        }
+      });
+    }
   } catch (error) {
-    console.error(error);
+    // On error, enable all buttons as a fallback to maintain current behavior
+    console.error("Error loading try_examples config:", error);
+    const buttons = document.getElementsByClassName(
+      "try_examples_button hidden",
+    );
+    for (let button of buttons) {
+      button.classList.remove("hidden");
+    }
   }
   tryExamplesConfigLoaded = true;
 };
 
 window.toggleTryExamplesButtons = () => {
   /* Toggle visibility of TryExamples buttons. For use in console for debug
    * purposes. */
-  var buttons = document.getElementsByClassName("try_examples_button");
+  var buttons = document.getElementsByClassName("try_examples_button hidden");
```

Comment:
This is a toggle function, so you need to be able to find toggle any button whether or not it is already hidden.

```suggestion
  var buttons = document.getElementsByClassName("try_examples_button");
```